Models/T2_relaxometry/mwf.m

```diff
@@ -188,7 +188,14 @@ function plotModel(obj, x, data, PlotSpectrum)
                 %----------------------- subplot 1 -----------------------%
                 subplot(2,1,1)
                 plot(T2.vals,Spectrum,'r');
-                title('Spectrums','FontSize',12);
+                %plot vertical bars at estimated T2
+                hold on
+                ax=gca;
+                hei=ax.YLim(2);hei=ceil(hei);
```

Comment:
@ileppe Handles are not object oriented in octave, and this line is not compatible with Octave. Travis crashes. 
use `hei = get(ax,'Ylim')` instead